tests/test_cerberus_client.py

```diff
@@ -569,3 +570,8 @@ def test_get_sdb_id_missing_id(self, mget):
         mget.return_value = self._mock_response(content=json.dumps(data))
         with self.assertRaises(CerberusClientException):
             self.client.get_sdb_id('not_found')
+
+    @patch('requests.get')
+    def test_request_headers_has_client_version(self, mget):
+        self.client.list_roles()
```

Comment:
I wrote a test with get_secrets_data() then I decided to go with the current design because:
1. They make calls in similar ways
2. We mainly use the client version for statistic analysis anyway, so testing with reading secret calls isn't strictly better for what we use it for
3. The test is going to be longer:
`    @patch('requests.get')
    def test_request_headers_has_client_version(self, mock_get):
        secret_data = {
            "data": {
                "sushi": "ikenohana",
                "ramen": "yuzu"
            }
        }

        mock_resp = self._mock_response(content=json.dumps(secret_data))
        mock_get.return_value = mock_resp
        self.client.get_secrets_data("fake/path", "ramen")
        mock_get.assert_called_with(ANY, headers=AnyDictWithKey('X-Cerberus-Client'), params=ANY)`